content/WebAuthn/WebAuthn_Developer_Guide/user-handle.adoc

```diff
@@ -0,0 +1,61 @@
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+
+
+== User Handle ==
+
+
+:toc:
+:toc-placement!:
+toc::[]
+
+This description is derived in large part from the W3C's recommendation, https://www.w3.org/TR/webauthn/[Web Authentication: An API for accessing Public Key Credentials Level 1].
+
+The user handle represents the mapping of a public key credential to a user account with the Relying Party (RP). As such, it is the RP that sets its value, ``user.id``.
+
+*Characteristics of ``user.id``*
+
+* Must not contain information that could identify the user.
+* Opaque byte sequence, maximum length = 64
+* Must be pseudo random
+* Must not start at 1 or 0
+
+An authenticator maps the user handle to the public key credential source. Authentication and authorization decisions are made on the basis of the ``user.id`` (not the user handle, which can be understood as a nickname in the sense of https://tools.ietf.org/html/rfc8266#section-6.1[`RFC 8266]).  The ``user.id`` and other user account attributes reside in the ``PublicKeyCredentialUserEntity`` dictionary, which is used when creating a new credential, as shown here:
+
+[source]
+----
+{
+   dictionary PublicKeyCredentialUserEntity : PublicKeyCredentialEntity {
+       required BufferSource   id;
+       required DOMString      displayName;
+   };
+}
+----
+
+The user handle is required for usernameless passwordless authentication with resident credentials, i.e., credentials that live on the security key. The example below illustrates how registration of the key creates the ``user.id`` underlying the user handle that is verified during authorization. In this example, at no point after registration is either a username or a password required.
```

Comment:
>The example below illustrates how registration of the key creates the ``user.id``

This might be understood as implying that the `user.id` is created along with the credential key pair, which is generally not the case. The `user.id` may be created when registering the first credential for an account, but in general it is a constant identifier stored in the user's account, and the same `user.id` value is used when registering any future credentials.